src/protocol_definitions.rs

```diff
@@ -383,7 +379,7 @@ impl UpdateOfferComponentInfoByte1 {
 /// LSB first Representation of OfferInformationComponentInfo
 pub struct OfferInformationComponentInfo {
     pub code: OfferInformationCodeValues,
-    pub reserved: u8,
+    _reserved: u8,
```

Comment:
Based on the fact that this struct is not `repr(C)`, and that there are `From` and that there are conversions to/from byte arrays directly, why do the reserved bytes need to be members of the Rust repr at all? The de/serialization methods can simply skip/pad the byte directly. Omitting them as members would mean the Rust type more directly represents the conceptual content, leading to more expressive and correct code.

Reply:
I would like to have those reserved fields are kept in the structures to match the protocol definitions, and make them private and not accessible. On the wire, these values are kept as 0.